base/docs/helpdb/Base.jl

```diff
@@ -2603,7 +2603,8 @@ creates a task, and does not run it.
 """
     fld(x, y)
 
-Largest integer less than or equal to `x/y`.
+Largest integer `n` such that `y * n` is less than or equal to `x`. Note that this can be
```

Comment:
I am under the impression that @yuyichao was saying (in that comment) that the documentation should change to reflect the behavior of the function. So, I changed the docs appropriately.